### ISSUE TITLE
Mobile behavior hover

### ISSUE THREAD

**Issue by romanlex** [gilbarbara/react-tooltips#25]
Example: On mobile phones when event is 'hover' and tooltip is a button what opening modal window
I'm click on this button for opening modal and I have active tooltip.

**Comment by gilbarbara** [comment]
Sorry, I can't understand what you are trying to say.

**Comment by gilbarbara** [comment]
Are you trying to have a `hover` tooltip on an element that already has a click event? is that it?

If it is, you should use the `target` prop and add the tooltip without children "after" your button as showed in the demo.

```jsx
<button className="modal-toggle" onClick={openModal}>Open Modal</button>
<Tooltip event="hover" target=".modal-toggle" content="This is the modal" />
```

**Comment by romanlex** [comment]
> Are you trying to have a hover tooltip on an element that already has a click event?

Yes, you right

> If it is, you should use the target prop and add the tooltip without children "after" your button as showed in the demo.

it's doesn't work https://codesandbox.io/s/vnx47lon7y

**Comment by gilbarbara** [comment]
ok, I'll take a look later

**Comment by gilbarbara** [comment]
Sorry, I gave you a wrong answer before...
You can only use the tooltip without children in `controlled` mode.

I don't think adding event listeners to foreign elements like `target` is a good idea so I ended adding a 	`disableHoverToClick` prop that will prevent tooltips with *hover* type to use *click* on mobile. b94e7fcdde8433137b083075f07e2619008d5d85

It's published as 0.4.0.

If you need better control in your app you can use the controlled mode.
